### PR TITLE
chore: setup foundation for vuemontreal 2.1

### PR DESCRIPTION
**Pull request by f3ltron**
### PR check list

- [x] The title of the PR is formatted following [conventional commits](https://www.conventionalcommits.org/) format 
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Functionality has been verified and is ready to be code reviewed

## What is the new behavior?

- I added base tailwind configuration for the new design.
- I added [nuxt-stories](https://github.com/richardeschloss/nuxt-stories) that let you create story. You can access to them as describe in the readme
- I have edited the basic template given by nuxt story to have something a bit better

`localhost:3000/.stories`

It's not publish in production.

**Does this PR introduce a breaking change?** 
 No

It doesnt break what existing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,16 @@
...
-
 # Check our projects if you want to know more about what next for vue montreal
+
+## Dev
+
+`npm ci && npm run dev`
+
+go to [localhost](http://localhost:3000)
+
+you can have access to our [stories](http://localhost:3000/.stories)
```
Comment (tlebel) [inline README.md:26]: Est-ce que https://deploy-preview-157--vuemontreal.netlify.app/.stories  devrait fonctionner?

Comment (f3ltron) [reply README.md:26]: non c'est normal la 404

**Comment by tlebel** [comment]
I don't understand why but when I run the branch, do `npm ci` I end up with these untracked files:  



then if I do `npm run dev` I get this error:

image

I have to delete those files to get it working. 

We should fix that because it will have to be repeated every time we `npm ci`.

File: README.md [hunk 2]
```diff
@@ -11,9 +11,16 @@
 - [Stephane Reiss](https://github.com/T0RAT0RA)
 - [Jeremie Bardon](https://github.com/jeremiebardon)
 
-## Development
+## Satus
```
Comment (tlebel) [inline README.md:14]:
```suggestion
## Status
```

File: README.md (hunk 1 above)
Comment (tlebel) [inline README.md:26]: Just to make things a bit clearer/cleaner: 

```suggestion
Go to: [http://localhost:3000](http://localhost:3000)

You can access our stories at: [http://localhost:3000/.stories](http://localhost:3000/.stories)
```

Comment (tlebel) [reply README.md:26]: Ok, I see here: https://github.com/richardeschloss/nuxt-stories#configuration

We would have to set `forceBuild: true` to get those on netlify.

Would it be useful to have this ran on something like https://www.chromaticqa.com/ for visual regressions?

Comment (f3ltron) [reply README.md:26]: we can create a new ticket for adding / merge to chroma yeah. They have a free plan now

Comment (f3ltron) [reply README.md:26]: yeap exactly.
About the file generate we have to figure out i didn't notice about the files generation. It's a little plugin that someone did maybe not the best one but i like the idea to not have two configs. One for project and one for storybook

**Comment by f3ltron** [comment]
Maybe i can delete nuxt stories and keep the docs and watch for chroma insted. But merge the tailwind config

**Pull request closed without merging**